mongodbatlas/provider_test.go

```diff
@@ -249,3 +255,19 @@ func testCheckPrivateEndpointServiceDataFederationOnlineArchiveRun(tb testing.TB
 		tb.Skip("`MONGODB_ATLAS_PRIVATE_ENDPOINT_ID` must be set for Private Endpoint Service Data Federation and Online Archive acceptance testing")
 	}
 }
+
+var _ plancheck.PlanCheck = debugPlan{}
+
+type debugPlan struct{}
+
+func (e debugPlan) CheckPlan(ctx context.Context, req plancheck.CheckPlanRequest, resp *plancheck.CheckPlanResponse) {
```

Comment:
we could extract these acceptance test utilities (including all the `testCheck*` functions) currently defined in `provider_test.go` into a separate package like `./mongodbatlas/acctest/utils.go`.

Reply:
moved to `mongodbatlas/testutils`